src/abhishekSJun20/PrimeNumber.java

```diff
@@ -0,0 +1,66 @@
+package abhishekSJun20;
+
+/*
+ 1: Create a class to satisfy below requirements. 
+    a) Given number is prime or not. 
+    b) Print total count of prime numbers between 2->100.
+    c) Find sum of all prime numbers between 2->100
+    d) Find avg of prime numbers between 2->100
+
+ */
+public class PrimeNumber {
+
+	int primeNumber(int num) {
+		// Method to find if Number is Prime Number or not. It return num - If prime and
+		// returns 0 - If not Prime
+		for (int index = 2; index <= Math.sqrt(num); index++) {
+			if (num % index == 0) {
+				return 0;
+			}
+		}
+		return num;
```

Comment:
Where is the printing statement, number is prime or not?

Reply:
done changes as per comment